src/backend/ctrl_flow_struct/graph_utils/mod.rs

```diff
@@ -0,0 +1,226 @@
+//! Various graph algorithms
```

Comment:
This module seems to be a building block for CF restructuring. Please add some tests for this.

Reply:
`graph_utils/tests.rs` ?